index.html

```diff
@@ -9,6 +9,35 @@
 </head>
 
 <body>
+    <header>
+        <nav class="navegation">
+            <div class="logo">
+                <img src="images/logo 1.svg" alt="logo">
+            </div>
```

Comment:
É melhor colocar uma classe do elemento pai como `logo-wrap` e colocar a classe `logo` na imagem assim vc pode evitar usar o elemento `img` como seletor no CSS, tipo assim:
```CSS
.logo-wrap .logo {}
```


```suggestion
            <div class="logo-wrap">
                <img src="images/logo 1.svg" alt="logo" class="logo">
            </div>
```